android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/ServiceCard.kt

```diff
@@ -190,14 +191,14 @@ private fun RowScope.RenderDetails(
         )
       }
     }
-    // Display 2 icons and counter if icons are more than 2
-    if (twoMemberIcons.isNotEmpty()) {
+    // Display N icons and counter if icons are more than N
+    if (nMemberIcons.isNotEmpty()) {
```

Comment:
According to this [comment](https://github.com/opensrp/fhircore/issues/2867#issuecomment-1827562566) we can only show 2 icons if 3 won't fit.

Won't this change remove the upper restriction on how many icons can be displayed?

Reply:
Based on Henry's comment here, the limit is 3.

> FYI: Only show number counter if we have more than 3 icons, if we have only 3 icons then counter should be replaced with the 3rd icon.

Reply:
The counter still needs to be displayed if there are more than 3 icons.

Reply:
If 3 icons can fit well, we can update this NUMBER_OF_ICONS_DISPLAYED to 3 .

Reply:
Initially, there was a limit of displaying 2 icons plus a number showing how many more icons haven't been displayed. I changed it to nMemberIcons for flexibility and replaced the hardcoded 2s with const val NUMBER_OF_ICONS_DISPLAYED.

Now since that same render function is used on the household register and household profile, that `n` limit is enforced on both screens. As a result, the household profile behaves the same as the register and allows a max of `n` icons as well. But according to the designs, the household profile shouldn't show more than one icon even if more than one icon applies to a patient